builtin/aws/ecr/registry.go

```diff
@@ -154,14 +152,13 @@ func (r *Registry) setupRepo(
 func (r *Registry) AccessInfo(
 	ctx context.Context,
 	log hclog.Logger,
-	img *docker.Image,
```

Comment:
We discovered that the ECR registry will not have any `docker.Image` available to it, which actually prevented this method from ever being called. This method did not use the input either.